diff -urN d1x-rebirth_v0.56-src-unpatched/main/bmread.c d1x-rebirth_v0.56-src-patch/main/bmread.c
--- d1x-rebirth_v0.56-src-unpatched/main/bmread.c	2010-09-25 17:06:31.000000000 +0200
+++ d1x-rebirth_v0.56-src-patch/main/bmread.c	2011-04-17 20:13:01.871000061 +0200
@@ -285,7 +285,9 @@
 	char rawname[100];
 
 	if (skip) {
-		return piggy_register_sound( &bogus_sound, "bogus", 0 );
+		// We tell piggy_register_sound it's in the pig file, when in actual fact it's in no file
+		// This just tells piggy_close not to attempt to free it
+		return piggy_register_sound( &bogus_sound, "bogus", 1 );
 	}
 
 	removeext( filename, fname );
diff -urN d1x-rebirth_v0.56-src-unpatched/main/piggy.c d1x-rebirth_v0.56-src-patch/main/piggy.c
--- d1x-rebirth_v0.56-src-unpatched/main/piggy.c	2010-09-25 17:06:31.000000000 +0200
+++ d1x-rebirth_v0.56-src-patch/main/piggy.c	2011-04-17 20:13:01.872000061 +0200
@@ -211,6 +211,8 @@
 	if ( !in_file )	{
 		SoundOffset[Num_sound_files] = 0;	
 	}
+	else if (SoundOffset[Num_sound_files] == 0)
+		SoundOffset[Num_sound_files] = -1;		// make sure this sound's data is not individually freed
 
 	i = Num_sound_files;
    
