--- third_party/WebKit/WebKit/chromium/src/gtk/WebInputEventFactory.cpp	2010-06-29 09:46:49.000000000 +0100
+++ wief.new	2010-07-20 15:58:33.000000000 +0100
@@ -469,7 +469,7 @@
     // - Gtk makes the scroll amount a function of the size of the scroll bar,
     //   which is not available to us here.
     // Instead, we pick a number that empirically matches Firefox's behavior.
-    static const float scrollbarPixelsPerTick = 160.0f / 3.0f;
+    static const float scrollbarPixelsPerTick = 480.0f / 3.0f;
 
     switch (event->direction) {
     case GDK_SCROLL_UP:
