--- garglk/glk.h	2008-11-07 14:37:54.000000000 +0100
+++ garglk/glk-patched.h	2010-01-12 16:56:29.000000000 +0100
@@ -18,13 +18,13 @@
     32-bit unsigned integer type, and glsi32 to make sure it's really a
     32-bit signed integer type. If they're not, horrible things will happen. */
 #include <limits.h>
-#if (USHORT_MAX == 4294967295)
+#if (USHRT_MAX == 4294967295)
 typedef unsigned short glui32;
 typedef signed   short glsi32;
 #elif (UINT_MAX   == 4294967295)
 typedef unsigned int glui32;
 typedef signed   int glsi32;
-#elif (ULONG_MAX) == 4294967295)
+#elif (ULONG_MAX == 4294967295)
 typedef unsigned long glui32;
 typedef signed long glsi32;
 #else
--- terps/nitfol/glk.h	2008-11-07 14:37:54.000000000 +0100
+++ terps/nitfol/glk-patched.h	2010-01-12 16:56:29.000000000 +0100
@@ -18,13 +18,13 @@
     32-bit unsigned integer type, and glsi32 to make sure it's really a
     32-bit signed integer type. If they're not, horrible things will happen. */
 #include <limits.h>
-#if (USHORT_MAX == 4294967295)
+#if (USHRT_MAX == 4294967295)
 typedef unsigned short glui32;
 typedef signed   short glsi32;
 #elif (UINT_MAX   == 4294967295)
 typedef unsigned int glui32;
 typedef signed   int glsi32;
-#elif (ULONG_MAX) == 4294967295)
+#elif (ULONG_MAX == 4294967295)
 typedef unsigned long glui32;
 typedef signed long glsi32;
 #else
