--- kid3/id3form.cpp	2010-11-12 16:17:37.000000000 +0000
+++ kid3/id3form.cpp	2010-11-09 20:28:58.000000000 +0000
@@ -276 +276 @@
-	m_nameLabel = new QLabel(i18n("Name:"), m_fileWidget);
+	m_nameLabel = new QLabel(i18n("Filename:"), m_fileWidget);
@@ -283 +283 @@
-	QLabel* formatLabel = new QLabel(i18n("Format:") + QChar(0x2191),
+	QLabel* formatLabel = new QLabel(i18n("Set Filename from Tags:"),
@@ -294,3 +294 @@
-	QLabel* fromTagLabel = new QLabel(i18n("From:"), m_fileWidget);
-	fileLayout->addWidget(fromTagLabel, 1, 2);
-	m_fnV1Button = new QPushButton(i18n("Tag 1"), m_fileWidget);
+	m_fnV1Button = new QPushButton(i18n("from Tag 1"), m_fileWidget);
@@ -298 +296 @@
-	QPushButton* fnV2Button = new QPushButton(i18n("Tag 2"), m_fileWidget);
+	QPushButton* fnV2Button = new QPushButton(i18n("from Tag 2"), m_fileWidget);
@@ -301 +299 @@
-	QLabel* formatFromFilenameLabel = new QLabel(i18n("Format:") + QChar(0x2193),
+	QLabel* formatFromFilenameLabel = new QLabel(i18n("Set Tags from Filename:"), 
@@ -312,2 +309,0 @@
-	QLabel* toTagLabel = new QLabel(i18n("To:"), m_fileWidget);
-	fileLayout->addWidget(toTagLabel, 2, 2);
@@ -315 +311 @@
-		new QPushButton(i18n("Tag 1"), m_fileWidget);
+		new QPushButton(i18n("to Tag 1"), m_fileWidget);
@@ -318 +314 @@
-		new QPushButton(i18n("Tag 2"), m_fileWidget);
+		new QPushButton(i18n("to Tag 2"), m_fileWidget);
@@ -451 +447 @@
-	QLabel* formatLabel = new QLabel(i18n("Format:") + QChar(0x2191),
+	QLabel* formatLabel = new QLabel(i18n("Set Filename from Tags:"),
@@ -462,2 +457,0 @@
-	QLabel* fromTagLabel = new QLabel(i18n("From:"), m_fileWidget);
-	fileLayout->addWidget(fromTagLabel, 1, 2);
@@ -465 +459 @@
-	fileLayout->addWidget(m_fnV1Button, 1, 3);
+	fileLayout->addWidget(m_fnV1Button, 1, 2);
@@ -467 +461 @@
-	fileLayout->addWidget(fnV2Button, 1, 4);
+	fileLayout->addWidget(fnV2Button, 1, 3);
@@ -480,2 +473,0 @@
-	QLabel* toTagLabel = new QLabel(i18n("To:"), m_fileWidget);
-	fileLayout->addWidget(toTagLabel, 2, 2);
@@ -484 +476 @@
-	fileLayout->addWidget(m_toTagV1Button, 2, 3);
+	fileLayout->addWidget(m_toTagV1Button, 2, 2);
@@ -487 +479 @@
-	fileLayout->addWidget(toTagV2Button, 2, 4);
+	fileLayout->addWidget(toTagV2Button, 2, 3);
